mne/forward/_field_interpolation.py

```diff
@@ -391,11 +390,9 @@ def make_field_map(evoked, trans='auto', subject=None, subjects_dir=None,
     ----------
     evoked : Evoked | Epochs | Raw
         The measurement file. Need to have info attribute.
-    trans : str | 'auto' | None
-        The full path to the ``*-trans.fif`` file produced during
-        coregistration. If present or found using 'auto'
-        the maps will be in MRI coordinates.
-        If None, map for EEG data will not be available.
+    %(trans)s
+        "auto" (default) will load trans from the FreeSurfer directory
+        specified by ``subject`` and ``subjects_dir`` parameters.
```

Comment:
The documentation build is complaining about this docstring: https://app.circleci.com/pipelines/github/mne-tools/mne-python/15836/workflows/93a505d6-b34b-470c-bb9d-837df5c65498/jobs/47994?invite=true#step-139-1122

For now, it renders as:

```
trans : path-like | dict | instance of Transform | None
    If str, the path to the head<->MRI transform ``*-trans.fif`` file produced
    during coregistration. Can also be ``'fsaverage'`` to use the built-in
    fsaverage transformation.
    If trans is None, an identity matrix is assumed.

    .. versionchanged:: 0.19
       Support for 'fsaverage' argument.
    "auto" (default) will load trans from the FreeSurfer directory
    specified by ``subject`` and ``subjects_dir`` parameters.
```

Note, you can check the render in an IPython console with `make_field_map?`.

The `.. versionchanged:: 0.19` lines are in the dodict https://github.com/mne-tools/mne-python/blob/82a341076416bfbf3e62ae97e330ecfe9bedc1eb/mne/utils/docs.py#L3408-L3415

If you want to append text to this entry as you did, you have 2 options:
- remove `%(trans)s` and copy-back the docstring
- move the `.. versionchanged` note from the docdict to each docstring which uses `%(trans)s` (only 6 including `make_field_map`)

Reply:
Great - thank you, I've gone for the second option and hopefully pushed a fix..